sdk/python/src/sangria_sdk/_http.py

```
@@ -0,0 +1,57 @@
+            except ValueError:
+                body = {"error": response.text}
+            raise APIError(
+                message=f"Sangria backend error ({response.status_code})",
+                status_code=response.status_code,
+                payload=body,
+            )
+
+        try:
+            return response.json()
+        except ValueError as exc:
+            raise APIError("Sangria backend returned non-JSON response") from exc
```

Ruff (0.15.6)

[warning] Avoid specifying long messages outside the exception class

(TRY003)